packages/echo-base/src/module/verify.ts

```diff
@@ -1,11 +1,11 @@
-import ArgumentError from '../errors/ArgumentError';
-import { AppMetadata } from '../types/module';
+import { ModuleMetaData } from '../types/module';
 import { persistLocalModuleMeta } from './persist';
 
-export function verifyModulesMeta(modules: AppMetadata[]): AppMetadata[] {
-    if (modules.length === 0) {
-        throw new ArgumentError({ argumentName: 'No modules awaitable' });
+export function verifyModulesMeta(modules: ModuleMetaData[]): ModuleMetaData[] {
+    if (modules instanceof Array) {
+        modules.length > 0 && persistLocalModuleMeta('EchoModules', modules);
+        return modules;
     }
-    persistLocalModuleMeta('EchoModules', modules);
-    return modules;
+    console.log('No modules awaitable.');
```

Comment:
Console.log?

Reply:
ender til console.info() men velger å ha dem ettersom dette er greit å få med seg men potensielt aldri vil vises for bruker.